### PR TITLE
 add CUAV X7 and nora

### PR DESCRIPTION
**Pull request by cuhome**
add CUAV X7 and nora

### CODE REVIEW

**Comment by hamishwillee** [comment]
Please let me know when this is ready for review/merging and I'll give it another final review

**Comment by cuhome** [comment]
@hamishwillee I have checked it and it looks great, thank you.

File: en/flight_controller/README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ This list is not exhaustive - there are other compatible flight controllers and
 - [Manufacturer-Supported Autopilots](../flight_controller/autopilot_manufacturer_supported.md)
   - [AirMind MindPX](../flight_controller/mindpx.md)
   - [AirMind MindRacer](../flight_controller/mindracer.md)
+  - [CUAV X7](../flight_controller/cuav_x7.md)
```
Comment (hamishwillee) [inline en/flight_controller/README.md:24]: FYI, I removed a link to the cuav hardware repo as it was inconsistent. That info should be added (by you) if needed in the docs themselves.

**Comment by hamishwillee** [comment]
I've done a few more minor subedits :-). Merging now as I think this looks pretty solid.

**Pull request merged**